javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/core/resolution/Context.java

```diff
@@ -192,7 +192,7 @@
                 methodUsage = ((TypeVariableResolutionCapability) methodDeclaration)
                                       .resolveTypeVariables(this, argumentsTypes);
             } else {
-                throw new UnsupportedOperationException();
+                return Optional.empty();
```

Comment:
mmm, why a method declaration should not have the TypeVariableResolutionCapability? Is this ok?